client/my-sites/backup/date-picker.jsx

```diff
@@ -31,7 +31,7 @@ const DatePicker = ( { onSelectDate, selectedDate } ) => {
 	const firstKnownBackupAttempt = useFirstKnownBackupAttempt( siteId );
 	const oldestDateAvailable = useDateWithOffset(
 		firstKnownBackupAttempt.backupAttempt?.activityTs,
-		!! firstKnownBackupAttempt.backupAttempt
+		{ shouldExecute: !! firstKnownBackupAttempt.backupAttempt }
```

Comment:
Now `useDateWithOffset` accepts a config object so we need to pass `shouldExecute` inside of one.